Documentation/gettingstarted/identity-relevant-labels.rst

```diff
@@ -18,24 +18,41 @@ when included in evaluation, cause Cilium to generate a unique identity for each
+following exceptions:
 
-=================================== ==================================================
+================================== ==============================================
 Label                               Description
------------------------------------ --------------------------------------------------
-``k8s:io.kubernetes.pod.namespace`` Include all ``io.kubernetes.pod.namespace`` labels
-``k8s:app.kubernetes.io``           Include all ``app.kubernetes.io`` labels
-``k8s:!io.kubernetes``              Ignore all ``io.kubernetes`` labels
-``k8s:!kubernetes.io``              Ignore all other ``kubernetes.io`` labels
-``k8s:!beta.kubernetes.io``         Ignore all ``beta.kubernetes.io`` labels
-``k8s:!k8s.io``                     Ignore all ``k8s.io`` labels
-``k8s:!pod-template-generation``    Ignore all ``pod-template-generation`` labels
-``k8s:!pod-template-hash``          Ignore all ``pod-template-hash`` labels
-``k8s:!controller-revision-hash``   Ignore all ``controller-revision-hash`` labels
-``k8s:!annotation.*``               Ignore all ``annotation labels``
-``k8s:!etcd_node``                  Ignore all ``etcd_node`` labels
-=================================== ==================================================
-
+---------------------------------- ----------------------------------------------
+``any:!io.kubernetes``             Ignore all ``io.kubernetes`` labels
+``any:!kubernetes.io``             Ignore all other ``kubernetes.io`` labels
+``any:!beta.kubernetes.io``        Ignore all ``beta.kubernetes.io`` labels
+``any:!k8s.io``                    Ignore all ``k8s.io`` labels
+``any:!pod-template-generation``   Ignore all ``pod-template-generation`` labels
+``any:!pod-template-hash``         Ignore all ``pod-template-hash`` labels
+``any:!controller-revision-hash``  Ignore all ``controller-revision-hash`` labels
+``any:!annotation.*``              Ignore all ``annotation labels``
+``any:!etcd_node``                 Ignore all ``etcd_node`` labels
+================================== ==============================================
+
+The above label examples are all *exclusive labels*, that is to say they define
+which label keys should be ignored. These are identified by the presence of the
+``!`` character.
+
+Label configurations that do not contain the ``!`` character are *inclusive
+labels*. Once at least one inclusive label is added, only labels that match the
+inclusive label configuration may be considered relevant for identities.
+Additionally, when at least one inclusive label is configured, the following
+inclusive labels are automatically added to the configuration:
+
+====================================== =====================================================
+Label                                  Description
+-------------------------------------- -----------------------------------------------------
+``reserved:.*``                        Include all ``reserved`` labels
```

Comment:
That line can be dropped because https://github.com/cilium/cilium/pull/14114, which added it, wasn't backported to v1.7.